airbyte-db/db-lib/src/main/java/io/airbyte/db/PostgresUtils.java

```diff
@@ -33,7 +33,7 @@ public static Certificate getCertificate(final PostgreSQLContainer<?> container)
     container.execInContainer("su", "-c", "psql -U test -c \"ALTER USER postgres WITH SUPERUSER;\"");
 
     container.execInContainer("su", "-c", "openssl ecparam -name prime256v1 -genkey -noout -out ca.key");
-    container.execInContainer("su", "-c", "openssl req -new -x509 -sha256 -key ca.key -out ca.crt -subj \"/CN=localhost\"");
+    container.execInContainer("su", "-c", "openssl req -new -x509 -sha256 -key ca.key -out ca.crt -subj \"/CN=127.0.0.1\"");
```

Comment:
why this change is needed?

Reply:
as I understood from the documentation, it is more correct that the common name (CN) should be different for root certificates and server certificates. For cases like ours, it is recommended to use the IP address for the root certificate, and the localhost for the server certificate. These changes will not break the source or destination tests.